File: app/src/main/java/com/neome/java/api/ent/entMain/msg/MsgFormMappingResultGet.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entMain.msg;

import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.Types.MetaIdVar;
import com.neome.java.api.meta.base.Types.RowId;
import com.neome.java.api.meta.base.dto.FormValueRaw;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgFormMappingResultGet extends Msg
{
  @Nullable
  public RowId inputFormGridRowId;

  public MetaIdForm inputFormId;

  public FormValueRaw inputFormValueRaw;

  public MetaIdVar mappingVarId;

  @Nullable
  public RowId outputFormGridRowId;

  public MetaIdForm outputFormId;

  @Nullable
  public FormValueRaw outputFormValueRaw;
}

File: app/src/main/java/com/neome/api/ent/entMain/msg/MsgFormMappingResultGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entMain.msg

import com.neome.api.meta.base.dto.FormValueRaw
import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.nucleus.base.msg.Msg
import com.neome.api.meta.base.Types.RowId

interface MsgFormMappingResultGet : Msg
{
  val inputFormGridRowId: RowId?
  val inputFormId: MetaIdForm
  val inputFormValueRaw: FormValueRaw
  val mappingVarId: MetaIdVar
  val outputFormGridRowId: RowId?
  val outputFormId: MetaIdForm
  val outputFormValueRaw: FormValueRaw?
}